mcp_server/gitlab_tools.py

```diff
@@ -102,14 +195,18 @@ async def open_merge_request(
     description: Annotated[str, Field(description="MR description")],
     target: Annotated[str, Field(description="Target branch (in the original repository)")],
     source: Annotated[str, Field(description="Source branch (in the fork)")],
-) -> str:
+) -> Union[str | None, bool]:
```

Comment:
Even though we don't let models to run these tools directly, I think we should still honor best practices and this return type is rather confusing (and probably wrong, shouldn't it be a tuple?). I would suggest using a structured type here.